bin/setup_macOS

```diff
@@ -55,6 +55,7 @@ echo "--- Installing npm modules for Lab interface ---"
 if [ -d ./node_modules ]; then
   echo "Npm modules are already installed"
 else
+  brew install yarn
```

Comment:
this line already does it: https://github.com/kengz/Unity-Lab/blob/master/bin/setup_macOS#L51

Reply:
Hmmm. That's odd. I got an install error when I ran the macOS script "yarn command not found". This line fixed it.

Reply:
lul prolly sudo denied there. keep this then